internal/cli/install.go

```diff
@@ -511,6 +551,40 @@ func (c *InstallCommand) Flags() *flag.Sets {
 			Default: "waypoint",
 		})
 
+		f.StringVar(&flag.StringVar{
+			Name:    "cpu-request",
+			Target:  &c.config.CPURequest,
+			Usage:   "Configures the requested CPU amount for the Waypoint server in Kubernetes.",
+			Default: "100m",
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "mem-request",
+			Target:  &c.config.MemRequest,
+			Usage:   "Configures the requested memory amount for the Waypoint server in Kubernetes.",
+			Default: "256Mi",
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "storage-request",
```

Comment:
I think we should prefix these all with `-k8s` if that makes sense.

Reply:
Good point, done!